src/utils/logic-app/templateUtils.ts

```diff
@@ -87,32 +87,61 @@ export function generateTemplate(workflow: Workflow) {
     };
 }
 
-export function generateTemplateParameter(name: string) {
-    const value = normalizeResourceName(name);
-    const nameParameter = normalizeParameterName(`workflows_${name}_name`);
+export function generateTemplateParameter(workflow: Workflow) {
+    const value = normalizeResourceName(workflow.name!);
+    const nameParameter = normalizeParameterName(`workflows_${workflow.name}_name`);
+    const parameters =  { ...workflow.definition.parameters };
 
-    return {
+    const workflowNameParameters = {
         [nameParameter]: {
             value
         }
     };
-}
 
-export function generateTemplateParameterDefinition(name: string) {
-    const defaultValue = normalizeResourceName(name);
-    const nameParameter = normalizeParameterName(`workflows_${name}_name`);
+    for (const key of Object.keys(parameters)) {
+        // Rename parameters by appending `workflows_{workflow}` in front to avoid conflict of the same parameter name between multiple Logic Apps
```

Comment:
(nit) "prepending" instead of "appending in front"

Reply:
:)